themes/origins/snippets/test-collection.liquid

```diff
@@ -0,0 +1,38 @@
+<yc-slider
+  arrows
+  responsive
+  data-type="fit"
+  style="
+    --slider-gap: 15;
+    --slider-speed: 600;
+    --slider-height: auto;
+    --slider-per-page: 3;
+    --slider-breakpoint-lg: 2;
```

Comment:
These can be misleading as they refer to breakpoints (e.g 240px) and not max items per breakpoints, maybe something like `--slider-max-items-md`